### PR TITLE
adding changes to README from Terminal

### PR DESCRIPTION
**Pull request by robertdevlin-ms**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# training-github
\ No newline at end of file
+# training-github
+
+I will add new information from terminal
```
Comment (robertdevlin-ms) [inline README.md:3]: You are missing a period at the end of your sentence.

**Review by robertdevlin-ms — COMMENTED** [review]
Please review!!

**Comment by satonaoki** [comment]
OK!

**Comment by sc-ss** [comment]
test

**Review by aykhara — APPROVED** [review]
LGTM!

**Review by loubigard — APPROVED** [review]
LGTM ! Thank you for your update

**Comment by tx-kato-m** [comment]
OK

**Pull request merged**